index.html

```diff
@@ -8,7 +8,25 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <nav></nav>
-    <div class="main-container"></div>
+    <nav class="flex-div">
+        <div class="nav-left">
+            <h3 class="logo">My Logo</h3>
+        </div>
+        <div class="nav-right">
+            <img src="src/img/Union.png" alt="des-menu" class="menu">
+        </div>
+    </nav>
+    <section class="main-container">
+        <div class="greeting-text">
+            <h2 class="name">I'm Jhonatan.<br>Glad to see you!</h2>
```

Comment:
- [ ] Kindly consider taking an `h1` here. in all other headings in the whole webpage, you will be taking `h2 to h6`. we are urged to use h1 only for the main heading on each page to follow the best semantics and SEO practices.